src/components/shipments/ShipmentsTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { Delete } from '@material-ui/icons';
+import React from 'react';
+import { useQueryParams } from 'use-query-params';
+
+import {
+  DefaultQueryParams,
+  SuperMaterialTable,
```

Comment:
Is there any specific reason why you don't use the default exported `SuperMaterialTable`?